spec/requests/api/v1/keywords_controller_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V1::KeywordsController, type: :controller do
+  describe 'GET#index' do
+    it 'returns expected status' do
+      get :index
+
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'responds the valid keywords' do
```

Comment:
```suggestion
    it 'returns the valid keywords' do
```

(for consistency with the previous test — both information come from the `response` object ;) )